.circleci/main.yml

```diff
@@ -1635,7 +1651,143 @@ jobs:
+    executor: small-node
+    steps:
+      - checkout
+      - run: cd ./console/backend && npm ci
+      - run: cd ./console/backend && npm run lint:no-fix
+  console-lint-cypress:
+    executor: small-node
+    steps:
+      - checkout
+      - run: npm ci --prefix ./console/cypress
+      - run: npm run lint --prefix ./console/cypress
+  console-test-backend:
+    executor: xlarge-node
+    steps:
+      - checkout
+      - node-install-frontend-backend
+      - run: npm run test --prefix ./console/backend -- --coverage
+      - codecov/upload:
+          file: ./console/backend/coverage/lcov.info
+  console-test-frontend-components:
+    executor: xlarge-node
+    steps:
+      - checkout
+      - node-install-frontend-backend
+      - run: npm run test:components --prefix ./console/frontend
+  console-test-frontend-react:
+    executor: xlarge-node
+    steps:
+      - checkout
+      - node-install-frontend-backend
+      - run: npm run test:frontend --prefix ./console/frontend -- --coverage
+      - codecov/upload:
+          file: ./console/frontend/src/coverage/lcov.info
+  console-build:
+    resource_class: small
+    docker:
+      - image: cimg/node:20.11.0
+    steps:
+      - checkout
+      - node-install-all
+      - run: cd console && make build
+  console-docker-build:
+    parameters:
+      resource_class:
+        type: string
+    resource_class: << parameters.resource_class >>
+    machine:
+      image: default
+    steps:
+      - checkout
+      - run:
+          name: docker login
+          command: echo "$DOCKER_PWD" | docker login --username pachydermbuildbot --password-stdin
+      - run:
+          name: Build application Docker image
+          command: make -C ./console circle-docker-build
+  console-docker-manifest:
+    machine:
+      image: default
+    steps:
+      - checkout
+      - run:
+          name: docker login
+          command: echo "$DOCKER_PWD" | docker login --username pachydermbuildbot --password-stdin
+      - run:
+          name: Create Multiarch Manifest
+          command: |
+            cd console
+            make circle-docker-manifest
+  console-test-docker-e2e-community-edition:
+    resource_class: xlarge
+    parallelism: 3
+    machine:
+      image: default
+    working_directory: ~/project/console
+    steps:
+      - checkout:
+          path: ~/project
+      - attach_workspace:
+          at: .
+      - run: npm ci
+      - install-bazel
+      - install-console-e2e-deps
+      - run:
+          name: Install pachd without auth
+          command: |
+            export CI=false # pachdev bug; CI=true assumes docker executor, not machine executor
+            export XDG_CONFIG_HOME=${PWD}/../etc/testing/console-test-helm-values
+            unset ENT_ACT_CODE # deactivate enterprise and auth
+            bazel run --config=remotecache --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} //src/testing/pachdev -- push --console=${CIRCLE_SHA1} -v
```

Comment:
We are able to use the built docker image we generate in CI with a bazel deploy by passing the `--console` flag to the bazel run command.